s3/container.go

```diff
@@ -79,7 +79,7 @@ func (c *container) Items(prefix, cursor string, count int) ([]stow.Item, string
 	// and correctly return the next list of items in subsequent requests.
 	marker := ""
 	if *response.IsTruncated {
-		marker = containerItems[len(containerItems)-1].Name()
+		marker = containerItems[len(containerItems)-2].Name()
```

Comment:
`Marker` starts from the current item
`StartAfter` starts from the one after this item